src/main/java/fi/helsinki/cs/tmc/core/domain/Exercise.java

```diff
@@ -272,6 +274,26 @@ public void setAdaptive(boolean adaptive) {
         isAdaptive = adaptive;
     }
 
+    public int getWeek() {
+        return week;
+    }
+
+    public void setWeek(int week) {
+        this.week = week;
+    }
+
+    public void generateWeek() {
+        String weekWord = "viikko";
```

Comment:
This seems super fragile. Remember that we are also providing courses in English. I understand that the initial roll-out will be to Finnish language courses only, but I'm still not sure if this is an assumption we can reasonably make. At the same time, I don't have any significantly less fragile alternative in mind. Opinions, @nygrenh ?